### ISSUE TITLE
Rb Buff Suggestions 

### ISSUE THREAD

**Issue by Provolonage** [captaincobbs/ESG-Mod#1157]
Buff the sterile/temperature tag Indy and Science on Rb's unique PPP and Xeno-Indy to +10 from +5, since you cant be hot and cold at the same time these buildings already scale worse since they cant both give +30 per planet.

Give Rb a unique version of Punctuated Evolution Foundation, found on Climate Engineering, that replaces the fertile tag to sterile, but make it give either 2-4 science per pop instead of 5 because this could snowball very hard, we can buff it to five it it feels necessary.

This is probably a bit more controversial but adding +1 pop slot +8 approval per pop on gas to the Biophobic trait, it makes sense from a lore perspective and game play wise as a buff, I cannot think of any planet more hostile to life then gas giants.

Very spicy take, Rb's start is really hard to work with, the desert does not give enough indy early to support decent expansion tempo and any actual investment in the system, you are forced to get your hero to level 2 put them on the home system then spam scouts and colonizers till turn 12-14, usually with only 4 pop by then. Once you don't need to focus on expansion the home system chain gangs, hopefully only one Epistis, prints some pop then ships the pop off and make singularities because putting pop on the tundra and forest are such a waste for the faction, with such bad planets overall and 0% chance of any resources that could synergize with other improvements the rb home system is essentially a two planet system with a desert and a really good snow, treating it like a "real" home system is a mistake. If other factions get super strong t0's such as oceans/forests/terrans, I think its fair that Rb gets an Ash instead of a desert to help with early indy issues. This still leaves the issue of Rb having very few pop slots they "want" to fill on the home system as the tundra only gives 6 science and 3 dust per pop if you aren't using the food, which you really shouldn't. 

Issues that I don't have a fix/suggestion for:

Signal curios being landmines for the faction is very annoying to get used to and kinda sad that Rb's gameplay is optimized against food meaning that investment in food and minors is not rewarding enough for the investment.

Rb questline has many very slow paths that aren't worth going down, but the very fast paths basically don't have rewards. The Scrap Kings law, +2 levels on ships and +35% indy cost on ships is just bad, if its going to take a law slot I think it should be +1 level and -5% cost as Rb's ships are already expensive due to the high slot count and to avoid the easy workaround of building empty hulls then retrofitting them. The hero reward is pretty weak and is actually quite slow to complete. 

With the guarantee of no luxs spawning on any home system, Rb is even less likely to have any access to t1 luxs with balanced start, without being forced to deliberately chase after them and if its a weaker t1 for Rb, such as transvine, dust trees, redsang or potatoes, its not cost effective to do such a thing, rushing Evaporation Inhibitors and doing sd2 with t2s is faster, cheaper and more rewards in most cases, it also has the added benefit of synergizing with arguably Rb's best quest reward, -50% pop construction cost, the requirement for being have 4 systems with a system dev level +1 higher then your highest at the moment of starting the quest, if you have 0 sd2's then its get 4 sd2's which is very easy to pull off if you delay sd2, which also feeds into the issue of its better to just kill off minors since sd2 is a good source of early food and infl and allows pop micro, by culling unnecessary pops you might be able to afford a t1 law till turn 20 and you don't need the food if you don't use it for anything, plus this reward makes printing pop very easy so having slightly less pop isn't a big issue since you will run out of pop slots on most systems with 10 turns anyway and since you are having at most 12 pop on the home system you always have a system after turn 20 sending extra pop around so even once you get the eventual +5 pop slots on sterile from multiple techs on the tech tree you generally aren't sitting around with far too many extra slots anyway.

**Comment by NoProblemWithThisAcc** [comment]
I'm fine with both building changes. I'm even fine with sterile gasses. I'm not sure about the ash either way.

Signal curios are just free Chain Gangs, right? But you're probably right about the quests: all options should be made viable.

Not sure on that last big paragraph.

**Comment by Provolonage** [comment]
Another idea is to nerf burrows to +1 pop slot on sterile from +2 and give RB +1 pop slot on sterile earlier, either to Biophobic, slap it onto all the quest rewards around step 3 or so, so every path gives +1 pop on sterile as well as the current rewards or to make the +1 pop slot on sterile empire passive that's found on the t3 empire tech give +2.

**Comment by wakystuf** [comment]
I think the last of these is likely resolved by the separate change to T2 lux spawns. The quest changes are incorporated as follows:

![image](https://user-images.githubusercontent.com/79942510/169659843-d300e213-b457-4aa5-a2bb-57d76f52bfb3.png)

**Comment by captaincobbs** [comment]
Can you summarize the changes?

**Comment by wakystuf** [comment]
Yes, it's simply the quest changes contained in the graphic I created

**Comment by captaincobbs** [comment]
kk
